lib/ssm_config.rb

```diff
@@ -3,11 +3,10 @@
 require 'ssm_config/ssm_storage/empty.rb'
 require 'ssm_config/errors.rb'
 require 'ssm_config/migration_helper.rb'
-require 'active_support/core_ext/hash/indifferent_access'
-require 'active_support/time'
+require 'active_support/all'
 
 module SsmConfig
-  VERSION = '1.3.3'.freeze
+  VERSION = '1.3.4'.freeze
   REFRESH_TIME = (30.minutes).freeze
```

Comment:
Do u folks want to change refresh time to an ENV variable ? This might be helpful against an ongoing issue with ssm_config

Reply:
@holmes2 thanks for the suggestion, we don't use the new features that you guys added so far on `ssm_config`. We might eventually, but we already have [SystemVariables](https://github.com/bodyshopbidsdotcom/snapsheet/blob/remove_gem_text/app/models/system_variable.rb#L13) which are solving similar issues.

Also, we should keep this PR only for Rails 7.

Thanks